src/configure.rs

```diff
@@ -109,7 +110,7 @@ fn get_editor_internal(visual: Option<OsString>, editor: Option<OsString>) -> Os
 fn get_config_path() -> OsString {
     let config_path = env::var_os("STARSHIP_CONFIG").unwrap_or_else(|| "".into());
     if config_path.is_empty() {
-        dirs::home_dir()
+        home_dir()
```

Comment:
I don't think we need to `use dirs_next::home_dir;` for a single call. I don't usually memorize the imports of a module before reading the code--this makes it harder to figure out where this function comes from, while making the overall length of the code longer.